README.md

```diff
@@ -26,79 +28,131 @@ NB: Ensure you have docker installed on your machine
   # start the app
   - `docker-compose up -d --build`
   # open on browser
-  - `http://127.0.0.1:8000/`
+  - `http://127.0.0.1:8000/` -->
+
+# Local Setup
 
-## USING virtualenv
-(Install virtualenv)
+## Install non-python dependencies
+
+1. **GDAL**
+
+On mac:
 ```bash
-$ pip install virtualenv
+$ brew install gdal
 ```
 
+2. **pango**
 
-# Create Virtualenv
+On mac:
 ```bash
-$ virtualenv --no-site-packages venv
+$ brew install pango
 ```
-* use no site packages to prevent virtualenv from seeing your global packages
-* . venv/bin/activate allows us to just use pip from command line by adding to the path rather then full path
 
-## Activate Virtualenv
+## Installing virtualenv
 ```bash
-$ source venv/bin/activate
+$ pip install virtualenv
 ```
 
+## Create virtualenv
+```bash
+$ virtualenv --no-site-packages myvirtualenvironmentname
+```
+* use no site packages to prevent virtualenv from seeing your global packages
+* . myvirtualenvironmentname/bin/activate allows us to just use pip from command line by adding to the path rather then full path
 
-## Fix probable mysql path issue (for mac)
-export PATH=$PATH:/usr/local/mysql/bin
-* or whatever path you have to your installed mysql_config file in the bin folder of mysql
+## Activate virtualenv
+```bash
+$ source myvirtualenvironmentname/bin/activate
+```
 
```

Comment:
Good working on this @shinroo , I would, however, like to suggest somethings that you can see if its good to include:
- Include a link to clone the repository as the first step of setup.
- An explanation of all the database configuration that must be in the .env file, we have not mentioned to the user that he should create a .env file.
- Make the ```myvirtualenvironmentname``` in this tags < myvirtualenvironmentname>to show this is a variable rather than a constant name.

Reply:
Hey @kelvinndmo thanks for the heads up - Andrew and I are sitting together and went through these and updated!